src/components/application_manager/src/commands/mobile/add_command_request.cc

```diff
@@ -95,31 +96,28 @@ void AddCommandRequest::Run() {
     }
   }
 
-  if (!((*message_)[strings::msg_params].keyExists(strings::cmd_id))) {
+  const smart_objects::SmartObject& msg_params = (*message_)[strings::msg_params];
+  if (!msg_params.keyExists(strings::cmd_id)) {
     LOG4CXX_ERROR(logger_, "INVALID_DATA");
     SendResponse(false, mobile_apis::Result::INVALID_DATA);
     return;
   }
 
-  if (app->FindCommand(
-          (*message_)[strings::msg_params][strings::cmd_id].asUInt())) {
+  if (app->FindCommand(msg_params[strings::cmd_id].asUInt())) {
     LOG4CXX_ERROR(logger_, "INVALID_ID");
     SendResponse(false, mobile_apis::Result::INVALID_ID);
     return;
   }
 
   bool data_exist = false;
 
-  if ((*message_)[strings::msg_params].keyExists(strings::menu_params)) {
+  if (msg_params.keyExists(strings::menu_params)) {
```

Comment:
@EKuliiev please extract refactoring changes to separate commit

Reply:
agree. done.